package.json

```diff
@@ -10,6 +10,7 @@
   "main": "dist-js/app.js",
   "scripts": {
     "postinstall": "electron-builder install-app-deps",
+    "postversion": "git push && git push --tags",
```

Comment:
I'd like to keep these kind of things up to the developer, so in this case it'd better if you set up an alias on your machine, because it can be handy also for other repos.